dvc/external_repo.py

```diff
@@ -15,15 +15,16 @@
 from dvc.exceptions import FileMissingError, PathMissingError
 from dvc.remote import RemoteConfig
 from dvc.utils.fs import remove, fs_copy
-from dvc.scm import SCM
+from dvc.scm.git import Git
 
 
 logger = logging.getLogger(__name__)
 
 
 @contextmanager
-def external_repo(url, rev=None):
-    path = _cached_clone(url, rev)
+def external_repo(url, rev=None, for_write=False):
+    logger.info("Creating external repo {}@{}", url, rev)
```

Comment:
I don't think it should be at info level. Also it would say `Creating external repo url@None`, which is confusing and ugly :slightly_smiling_face:

Reply:
Yeah, I meant this to be a debug one.